account/user_blackbox_test.go

```diff
@@ -58,6 +59,49 @@ func (s *userBlackBoxTest) TestOKToLoad() {
 	createAndLoadUser(s) // this function does the needful already
 }
 
+func (s *userBlackBoxTest) TestLoadByUsername() {
+	t := s.T()
+	resource.Require(t, resource.Database)
+	t.Run("load ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1, func(fixture *tf.TestFixture, idx int) error {
+			fixture.Identities[idx].Username = "myusername"
+			return nil
+		}), tf.Users(1))
+		loadedUser, err := s.repo.LoadByUsername(s.Ctx, "myusername")
+		require.NoError(t, err, "Could not load user")
+		require.Equal(t, loadedUser[0].Email, fxt.Users[0].Email)
+		require.Equal(t, loadedUser[0].ID, fxt.Users[0].ID)
+	})
+	t.Run("load one user with a list of identities associated to a this user", func(t *testing.T) {
+		random := uuid.NewV4()
+		myusername := "myusername" + random.String()
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.Users(1), tf.Identities(5, func(fixture *tf.TestFixture, idx int) error {
```

Comment:
You must use `t` instead of `s.T()` because `s.T()` refers to the top level test and not the nested one.

```suggestion
		fxt := tf.NewTestFixture(t, s.DB, tf.Users(1), tf.Identities(5, func(fixture *tf.TestFixture, idx int) error {
```

Reply:
fixed in f58c3433f95de90ec2fe998ed6ea1eeb1d6564f5